core/src/main/java/com/linecorp/armeria/internal/server/DefaultServiceRequestContext.java

```diff
@@ -116,6 +116,7 @@
 
     @Nullable
     private String strVal;
+    private boolean shouldReportUnLoggedException;
```

Comment:
`UnLogged` -> `Unlogged`?

Reply:
All `UnLog` is changed into `Unlog`. I think unlog is the right word 😃 .